docs/reference/manifest/appdomains.md

```diff
@@ -13,6 +13,9 @@ Lists any domains in addition to the domain specified in the SourceLocation elem
 </AppDomains>
 ```
 
+> [!IMPORTANT]
+> Each **AppDomain** must include the protocol e.g., `<AppDomain>https://myappdomain"<AppDomain>`.
```

Comment:
Suggest rephrasing to -> The value of each **AppDomain** element must include the protocol (e.g., `<AppDomain>https://myappdomain<AppDomain>`).

- puts "e.g.,..." in parenthesis
- removes the erroneous quotation mark from the end of the specified domain